### ISSUE TITLE
ROCm Crossfire Integration

### ISSUE THREAD

**Issue by workspace-public** [Xilinx/RapidWright#922]
I have filed this issue also under ROCm feature requests; this is a request for the "right department." Is it possible to create a PCIe FPGA that converts crossfire capable GPU into something compatible with INSTINCT platform-specific features? Where would anyone get the proprietary blobs and API reference?

**Comment by clavin-xlnx** [comment]
That is an interesting thought and question.  Can you explain more about what your ultimately goal is?  

It sounds like you want to be able to use older Crossfire-capable GPUs with ROCm, is that correct?

**Comment by clavin-xlnx** [comment]
I don't believe there exists any FPGA designs or IP that can achieve what you are proposing.  In theory, it could be possible, but the development effort would be significant and its not clear what the benefits would be.  Can you share your motivation or more details on the problem you are trying to solve?

**Comment by clavin-xlnx** [comment]
Closing as this appears resolved.  Please re-open or file a new issue if there are additional concerns.
